src/wwwroot/app/src/scripts/__ClassroomJoin.js

```diff
@@ -0,0 +1,105 @@
+var tempsid = $('#tempsid').val();
+
+var colorArray = [
+    "red darken-1", "teal darken-1", "indigo darken-1", "blue-grey darken-1",
+    "purple darken-3", "blue lighten-1", "blue accent-2", "light-blue"
+];
+
+$(document).ready(function () {
+    getClassRoom();
```

Comment:
we already use this method for teachers. getClassRoom();
you can try to use getClassRoomForStudent();